tools/exporters/data_dictionary_builder.php

```diff
@@ -193,36 +196,41 @@
                 continue;
             }
 
-            print "Inserting $table $bits[1]\n";
+            print "\tInserting $table $bits[1]\n";
             $parameterCount++;
             $bits[2] = htmlspecialchars($bits[2]);
             //find values to insert
             $Name = $table . "_" . $bits[1];
 
             $query_params = array(
-                             'Name'            => $Name,
-                             'Type'            => $bits[0],
-                             'Description'     => $bits[2],
-                             'SourceField'     => $bits[1],
-                             'SourceFrom'      => $table,
-                             'CurrentGUITable' => 'quat_table_'
```

Comment:
@driusan this was causing issues, is it being used anywhere ????

Reply:
the quat table part? Yeah, you can remove that..